solidity/ecdsa/test/fixtures/index.ts

```diff
@@ -60,7 +60,7 @@ export const walletRegistryFixture = deployments.createFixture(
     operators: Operator[]
     reimbursementPool: ReimbursementPool
   }> => {
-    await deployments.fixture(["WalletRegistry"])
+    await deployments.fixture(["WalletRegistryTransferGovernance"])
```

Comment:
This will execute all deployments steps up to 08_, included. Correct? If so, why don't we execute up to 09_, included?

Reply:
We can actually skip a tag reference and just do `await deployments.fixture()` to get a snapshot of the whole deployment.

Reply:
ae8e9f6dd680a5cf1d89206485fd3d0b6d0e31b4